apps/bfd-model/bfd-model-dsl-codegen/bfd-model-dsl-codegen-plugin/src/test/java/gov/cms/model/dsl/codegen/plugin/transformer/ShortStringFieldTransformerTest.java

```diff
@@ -0,0 +1,64 @@
+package gov.cms.model.dsl.codegen.plugin.transformer;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.squareup.javapoet.CodeBlock;
+import gov.cms.model.dsl.codegen.plugin.accessor.GrpcGetter;
+import gov.cms.model.dsl.codegen.plugin.accessor.StandardSetter;
+import gov.cms.model.dsl.codegen.plugin.model.ColumnBean;
+import gov.cms.model.dsl.codegen.plugin.model.MappingBean;
+import gov.cms.model.dsl.codegen.plugin.model.TransformationBean;
+import org.junit.jupiter.api.Test;
+
+/** Unit test for {@link ShortStringFieldTransformer}. */
+public class ShortStringFieldTransformerTest {
+  /** Verifies that required fields use {@code copyShortString}. */
+  @Test
+  public void testRequiredField() {
+    ColumnBean column =
+        ColumnBean.builder().name("idrDtlCnt").nullable(true).sqlType("char(1)").build();
+    TransformationBean transformation =
+        TransformationBean.builder()
+            .optionalComponents(TransformationBean.OptionalComponents.None)
+            .from("idrDtlCnt")
+            .build();
+    MappingBean mapping =
+        MappingBean.builder()
+            .entityClassName("gov.cms.test.Entity")
+            .transformation(transformation)
+            .build();
+
+    ShortStringFieldTransformer generator = new ShortStringFieldTransformer();
+    CodeBlock block =
+        generator.generateCodeBlock(
+            mapping, column, transformation, GrpcGetter.Instance, StandardSetter.Instance);
+    assertEquals(
+        "transformer.copyShortString(namePrefix + gov.cms.test.Entity.Fields.idrDtlCnt, true, from.getIdrDtlCnt(), to::setIdrDtlCnt);\n",
+        block.toString());
```

Comment:
I'd wonder if this isn't a bit brittle; perhaps we'd be better just checking that "copyShortString" exists within the generated code block?

Reply:
Just verifying the string is generated correctly.  Same as `assertEquals(1)` would be for an integer field.